modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/frontend/taglib/servlet/taglib/ServerPortalPropertiesScreenNavigationCategory.java

```diff
@@ -15,48 +15,33 @@
 package com.liferay.server.admin.web.internal.frontend.taglib.servlet.taglib;
 
 import com.liferay.frontend.taglib.servlet.taglib.ScreenNavigationCategory;
-import com.liferay.frontend.taglib.servlet.taglib.ScreenNavigationEntry;
-import com.liferay.frontend.taglib.servlet.taglib.util.JSPRenderer;
 import com.liferay.portal.kernel.language.Language;
 import com.liferay.server.admin.web.internal.constants.ServerAdminNavigationEntryConstants;
 
-import java.io.IOException;
-
 import java.util.Locale;
 
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Albert Lee
  */
 @Component(
-	property = {
-		"screen.navigation.category.order:Integer=20",
-		"screen.navigation.entry.order:Integer=10"
-	},
-	service = {ScreenNavigationCategory.class, ScreenNavigationEntry.class}
+	property = "screen.navigation.category.order:Integer=20",
+	service = ScreenNavigationCategory.class
 )
 public class ServerPortalPropertiesScreenNavigationCategory
-	implements ScreenNavigationCategory, ScreenNavigationEntry<Object> {
+	implements ScreenNavigationCategory {
 
 	@Override
 	public String getCategoryKey() {
 		return ServerAdminNavigationEntryConstants.
 			CATEGORY_KEY_PORTAL_PROPERTIES;
 	}
 
-	@Override
-	public String getEntryKey() {
-		return ServerAdminNavigationEntryConstants.ENTRY_KEY_PORTAL_PROPERTIES;
```

Comment:
@MrJohn1911 There's still a usage to this constant. To make it safe, if we choose to delegate, we'd better remove the `ENTRY_KEY_` constant, replacing all its usages with the `CATEGORY_KEY_` one.

Reply:
@dantewang So, you want me to do this in this specific case or to all `ENTRY_KEY` constants that I stopped using in my pull requests?

Reply:
@MrJohn1911 OK, let's only do the delegate (`getEntryKey() -> getCategoryKey()`) when both methods are using the same constant, not when the constants have the same content/value.